### ISSUE TITLE
can you provide a c++ using interface？

### ISSUE THREAD

**Issue by IvyGongoogle** [apoorvavinod/Real_time_Object_detection_and_tracking#15]
Hello, I see you give a `main.py` python script about how to detect and track, but can you provide a c++ using interface？

Looking forward your reply.

**Comment by apoorvavinod** [comment]
I would, but converting my code to c++ would take some time. Besides there should be already be other C++ implementations of tracking/detection if you search on github.

**Comment by IvyGongoogle** [comment]
@apoorvavinod  Thanks for your reply. I can find some `tracking/detection` c++ repos in github using `kcf`, but they all is for specific object, such as person, face or vehicle. I am confused that the total pipeline of your code is same with these `tracking/detection` methods about specific object?

**Comment by apoorvavinod** [comment]
Yes. The tracking is for specific objects detected by SSD or YoloV2 models. For example the YoloV2 weights I am using is trained to detect these classes: https://github.com/apoorvavinod/Real_time_Object_detection_and_tracking/blob/master/model_data/coco_classes.txt
